### PR TITLE
add a task to open a database console

### PR DESCRIPTION
**Pull request by djordjelacmanovic**
Added support for running `rails dbconsole`, which opens an interactive command line interface to the app's database.

### CODE REVIEW

File: tasks/mina/rails.rb [hunk 1]
```diff
@@ -19,6 +19,14 @@
...
 end
 
+desc 'Starts an interactive database console'
+task :dbconsole do
+  set :execution_mode, :exec
+  in_path "#{fetch(:current_path)}" do
+    command %{#{fetch(:rails)} dbconsole}
+  end
+end
+
```
Comment (d4be4st) [inline tasks/mina/rails.rb:29]: can you rewrite it to use this template:
https://github.com/mina-deploy/mina/pull/672/files#diff-fe6c80d1b3934ea10fe1bc738791cda50ced69c723f18725fc87aba2077360ebR128

:)

**Review by d4be4st — CHANGES_REQUESTED** [review]

File: tasks/mina/rails.rb (hunk 1 above)
Comment (djordjelacmanovic) [reply tasks/mina/rails.rb:29]: The link is referring to this PR, did you intend to link elsewhere?

Comment (d4be4st) [reply tasks/mina/rails.rb:29]: Yes i did, chechk the line it points to.
At the end of the file

**Comment by lovro-bikic** [comment]
Thanks for the PR, however, we decided not to add this functionality to core. The reason is, we already support executing arbitrary Rails/Rake tasks, which you can invoke from CLI like this:
```bash
mina rails[dbconsole]
```
This will have the same effect as your proposed solution.

If you use this task often, you can also add it to your `config/deploy.rb`:
```ruby
task :dbconsole do
  invoke :rails, 'dbconsole'
end
```
which you can then invoke from CLI: `mina dbconsole`

You can do the same with Rake tasks: `mina rake[dbconsole]`

**Pull request closed without merging**